fdbclient/DatabaseConfiguration.cpp

```diff
@@ -659,6 +678,10 @@ bool DatabaseConfiguration::setInternal(KeyRef key, ValueRef value) {
 		blobGranulesEnabled = (type != 0);
 	} else if (ck == "encryption_at_rest_mode"_sr) {
 		encryptionAtRestMode = EncryptionAtRestMode::fromValueRef(Optional<ValueRef>(value));
+	} else if (ck.startsWith(storageEngineParamsPrefix.removePrefix(configKeysPrefix))) {
+		// TODO : should we hardcode the above condition like others?
+		auto paramName = ck.removePrefix(storageEngineParamsPrefix.removePrefix(configKeysPrefix)).toString();
```

Comment:
Please add a comment for what this is doing.

Reply:
This is when you write the storage engine parameters to `DatabaseConfiguration`, it will insert the key/value pair into the _StorageEngineParamSet_, where all parameters' key/value pairs are saved in `DBConfig`

Reply:
I meant add a comment to the code.